### ISSUE TITLE
Audit PDF Print using test cases

### ISSUE THREAD

**Issue by michaelccata** [18F/e-QIP-prototype#1387]
## Where are you in the form?

Print PDF of the form - apparently some fields are not making it over or are blocked by layout changes.

More details to come.

## What did you see?

Screencaps TBD

- [ ] Validation error
- [ ] Data not saving
- [ ] Language or content issue
- [X] Other (explain below)

## What did you expect to see?

Full application viewable on the print PDF. 

## What operating system and browser/version are you using?

**Operating system**
- [X] Windows
- [ ] MacOS

**Browser**
- [ ] Chrome
- [ ] Firefox
- [ ] IE
- [ ] Edge
- [ ] Other

**Version:**

**Comment by christopherhuii** [comment]
@michaelccata @joshmfranklin do either of you have specific sections or screenshots i should be looking at? or should I just look at everything?

**Comment by joshmfranklin** [comment]
@christopherhuii I don't have specific sections. I believe the army said there were some missing labels or something along those lines but I don't believe we got any screenshots from them. @michaelccata do you have any more insight into what we should be looking for in the PDF?

**Comment by michaelccata** [comment]
Yeah, I think look at everything (formatting + data appearing where it should) is a good start.

**Comment by christopherhuii** [comment]
Without explicit details in this issue, I did a quick audit of the print PDF generated from test case 3.

I found a few issues. I am ONLY reporting the bugs that affect the integrity of form data (not styling and alignment issues):

1. Multiple countries
<img width="421" alt="screen shot 2019-02-06 at 12 35 33 pm" src="https://user-images.githubusercontent.com/8367504/52380067-2dab5000-2a21-11e9-8d00-06de3793156b.png">
This is a little unclear. We may want to find a better way to display multiple countries.

2. "$" overlapping the content of the input field
<img width="416" alt="screen shot 2019-02-06 at 12 51 06 pm" src="https://user-images.githubusercontent.com/8367504/52380053-2126f780-2a21-11e9-9107-25a8f31c64bd.png">

3. There were a few fields/questions that did not have data in them. After further investigation, I was able to determine that there are deeper issues in the form causing the print to be inaccurate.
These issues are: #1403 and #1404
